gammapy/scripts/image_pipe.py

```diff
@@ -2,56 +2,180 @@
+            self.header = self.empty_image.to_image_hdu().header
+        if bkg_image:
+            self.maps["total_bkg"] = bkg_image
+
+    def make_counts(self, obs_id):
+        """Fill the counts image for the events of one observation.
+
+        Parameters
+        ----------
+        obs_id : int
+            Number of the observation
+
+        """
+        self.maps["counts"] = SkyMap.empty_like(self.empty_image)
+        obs = self.data_store.obs(obs_id=obs_id)
+        events = obs.events.select_energy(self.energy_band)
+        events = events.select_offset(self.offset_band)
+        self.maps["counts"].fill(events=events)
+        self.maps["counts"].data = self.maps["counts"].data.value
         log.info('Making counts image ...')
 
-    def make_exposure(self):
-        log.info('Making exposure image ...')
+    def make_total_counts(self):
+        """Stack the total count from the observation in the 'DataStore'
+
+        """
+        self.maps["total_counts"] = SkyMap.empty_like(self.empty_image)
+        for obs_id in self.data_store.obs_table['OBS_ID']:
+            self.make_counts(obs_id)
+            self.maps["total_counts"].data += self.maps["counts"].data
+
+    def make_background(self, obs_id):
+        """Make the background map for one observation from a bkg model
+
+        Parameters
+        ----------
+        obs_id : int
+            Number of the observation
+
+        """
+        self.maps["bkg"] = SkyMap.empty_like(self.empty_image)
+        obs = self.data_store.obs(obs_id=obs_id)
+        table = obs.bkg.acceptance_curve_in_energy_band(energy_band=self.energy_band)
+        center = obs.pointing_radec.galactic
+        bkg_hdu = fill_acceptance_image(self.header, center, table["offset"], table["Acceptance"], self.offset_band[1])
+        livetime = obs.observation_live_time_duration
+        self.maps["bkg"].data = Quantity(bkg_hdu.data, table["Acceptance"].unit) * self.maps[
+            "bkg"].solid_angle() * livetime
+        self.maps["bkg"].data = self.maps["bkg"].data.decompose()
+        self.maps["bkg"].data = self.maps["bkg"].data.value
 
-    def make_background(self):
         log.info('Making background image ...')
 
+    def make_background_normalized_offcounts(self, obs_id, exclusion_mask):
+        """Normalized the background compare to te events in the counts maps outside the exclusion maps
+
+        Parameters
+        ----------
+        exclusion_mask : `~gammapy.image.ExclusionMask`
+            Exclusion regions
+
+
+        """
+        self.make_counts(obs_id)
+        self.make_background(obs_id)
+        counts_sum = np.sum(self.maps["counts"].data * exclusion_mask.data)
+        bkg_sum = np.sum(self.maps["bkg"].data * exclusion_mask.data)
+        scale = counts_sum / bkg_sum
+        self.maps["bkg"].data = scale * self.maps["bkg"].data
+        self.maps['exclusion'] = exclusion_mask
+
+    def make_total_bkg(self, exclusion_mask):
```

Comment:
Does  'make_background_normalized_offcounts' actually have to be visible to the user?
Because right now `make_total_bkg` only does the a loop over observation IDs. But if a list of obs IDs is passed, it can still contain only one value.